sql/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -927,4 +927,29 @@ public void testInsertGeneratedPrimaryKeyValueGiven() throws Exception {
         expectedException.expectMessage("Given value 0 for generated column does not match defined generated expression value 4");
         execute("insert into test(col1, col2) values (1, 0)");
     }
+
+    @Test
+    public void testInsertFromSubQueryMissingPrimaryKeyValues() throws Exception {
```

Comment:
can't move that, it's evaluated the RowShardResolver which is used by the ColumnIndexWriterProjector